<!DOCTYPE html><html lang="en"><head><meta name="generator" content="Hexo 3.9.0"><meta charset="UTF-8"><title>Project Summary</title><link rel="stylesheet" href="css/normalize.css"><link rel="stylesheet" href="/css/hexo-theme-adoubi.css"><link rel="icon" href="/images/favicon.ico"></head><body><div class="header"><a class="github-link"><img class="github-image" src="/images/Aquicon-Github-small.png"></a><a class="email-link" href="mailto:null"><img class="email-image" src="/images/email-small.png"></a><a class="subscribe-link" href="/atom.xml"><img class="subscribe-image" src="/images/subscribe-small.png"></a></div><div class="content"><div class="post-item"></div><h2 class="post-title-wrapper"><p class="post-title">Project Summary</p></h2><div class="post-date"><time datetime="2019-10-09T05:23:10.000Z">2019-10-09</time></div><div class="post-content"><h3 id="What-is-my-project"><a href="#What-is-my-project" class="headerlink" title="What is my project?"></a>What is my project?</h3><p>This project serves as an RNA-sequence-based Pan-Cancer Analysis which aims to examine the differences among the genomic alterations found across breast cancer using primarily proteomics dataset from The Cancer Genome Atlas(TCGA). The goal of the project is to identify differentially expressed genes and their correlated biological pathways based on the clustering of breast cancer subclasses, aside from the PAM50 test.</p>
<h3 id="What-are-my-research-target-and-research-method"><a href="#What-are-my-research-target-and-research-method" class="headerlink" title="What are my research target and research method?"></a>What are my research target and research method?</h3><p>Breast cancer is known for its diversity and the target of this project is to explore the underlying genes associated with the varieties in breast cancer and the uniqueness of each subtype.<br> Published iTRAQ proteome profiling of 77 breast cancer samples generated by the Clinical Proteomic Tumor Analysis Consortium(NCI/NIH) is included and analyzed to divide breast cancer patients into separate sub-classes. The data is clustered into separate clusters with K-Means clustering. The algorithm generates initial cluster centers arbitrarily so it is necessary to compare different numbers of clusters and run diagnostic checks. We checked 10 different values (k=1,2,3,4,5,6,7,8,9,10,20,79) for numbers of cluster centers and a random protein set is created and processed for comparison. The silhouette coefficient is used to measure the similarity of the data to their clustering configuration and the Homogeneity score is used to check whether the clusters contain only samples belonging to a single class.<br> The breast cancer sub-classes determined by K-Means clustering is compared with PAM50. The data is plotted by PCA to see which method gives a more robust clustering.<br> After the clusters are established, RNA-seq data from the same samples in the proteomics dataset from The Cancer Genome Atlas(TCGA), which contains the gene expression quantification data in transcriptome profiling, is used to identify differentially expressed genes in different subtypes. Then the up-genes are analyzed in DAVID which helps us to recognize the biological pathways correlated with the differentially expressed genes.</p>
<h3 id="What-was-the-biggest-challenge-that-I-encountered-How-did-I-overcome-it"><a href="#What-was-the-biggest-challenge-that-I-encountered-How-did-I-overcome-it" class="headerlink" title="What was the biggest challenge that I encountered? How did I overcome it?"></a>What was the biggest challenge that I encountered? How did I overcome it?</h3><p> I begin this project as a biology student who has just a very basic understanding of programming and statistics. In biology class, a gene is a concrete concept, commonly pictured as a colored fragment on a monochromatic X chromosome. However, during data processing, this vivid illustration faded as everything is converted to numbers and lines of coding. Gradually, I found it hard to interpret datapoint on the plots for I cannot relate a mere dot to the biological blueprint which codes for the characteristics of life. Also, the original data frame contains multi-dimensional information about the patients (age, genders…) but with PCA, data are compressed to two-dimension and computed on the axis with abstract naming of ‘principal component 1’ and ‘principal component 2’, making the interpretation of data points more confusing. </p>
<p> To understand the instructions given to the computer, it is necessary to read through the involved documents on scikit-learn. Still, it is difficult to understand or memorize all the languages at once so I tried to label parts of the coding that confuse me with their functions and translate the documents into intelligible language. I first went through each stage of data processing and then relate each step together to gain a holistic understanding of the programming. </p>
<p> Also, I reached diagrams and videos on conducting PCA to gain an overview of the whole process which is taken care of by a single line of coding from scikit-learn. It is quite fascinating to see all the datapoint being reflected on the axis and transform to a new direction, especially when the z-axis turns to be perpendicular to a surface and the points naturally project on a 2-D surface. With the assistance of explanatory GIFs, I finally get to comprehend and envision how the data is plotted two-dimensionally.</p>
<h3 id="What-was-the-most-interesting-part-of-my-project"><a href="#What-was-the-most-interesting-part-of-my-project" class="headerlink" title="What was the most interesting part of my project?"></a>What was the most interesting part of my project?</h3><p> In this project, we tried to define breast cancer subtypes in a different way other than the molecular subtypes defined by the PAM50 test which mainly focuses on hormone-receptors. With K-Means clustering, we classify the sample according to different features and explore the differences in different subcategories, which might lead to a more robust definition of breast cancer subtypes.  </p>
<h3 id="What-are-my-research-result-s-and-conclusion-s"><a href="#What-are-my-research-result-s-and-conclusion-s" class="headerlink" title="What are my research result(s) and conclusion(s)?"></a>What are my research result(s) and conclusion(s)?</h3><p>K-Means clustering: According to the comparison along 10 values for the number of clustering, k=3 gives a Silhouette Coefficient of 0.1069 and the Homogeneity score jumps 2-fold with a value of 0.0734, being the closest value to 1.0. (Except for k=79, which gives a Homogeneity score of 1.0, but this clustering is worthless as every protein is clustered as a separate individual.) Hence, cluster number k=3 is used to compare with PAM50 subtypes.</p>
<p>Shown by PCA plots for proteomics and RNA data, clustering by PAM50 cannot be distinctly identified whereas clustering by K-Means when k=3 gives quite distinct subgroups. K-Means clustering number k=4 also gives a relatively ideal clustering but one cluster contains only one data point, so clusters determined by k=3 are used in gene expression analysis. </p>
<p>The results of gene expression analysis are shown in the data frame.</p>
<h3 id="PAM50"><a href="#PAM50" class="headerlink" title="PAM50"></a><center>PAM50</center></h3><p>Fructose and mannose metabolism</p>
<p><img src="/2019/10/09/Project-Summary/1.png" alt="1.png"></p>
<p>Amino sugar and nucleotide sugar metabolism</p>
<h3 id="Basal-like"><a href="#Basal-like" class="headerlink" title="Basal-like"></a>Basal-like</h3><table>
<thead>
<tr>
<th>Category</th>
<th>Term</th>
<th>Count</th>
<th>%</th>
<th>P Value</th>
</tr>
</thead>
<tbody><tr>
<td>KEGG_PATHWAY</td>
<td>hsa00051:Fructose and mannose metabolism</td>
<td>5</td>
<td>1.089324619</td>
<td>0.00834434</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa00520:Amino sugar and nucleotide sugar metabolism</td>
<td>5</td>
<td>1.089324619</td>
<td>0.033038389</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa04360:Axon guidance</td>
<td>8</td>
<td>1.74291939</td>
<td>0.042729458</td>
</tr>
</tbody></table>
<p><img src="/2019/10/09/Project-Summary/2.png" alt="1.png"></p>
<p>Axon guidance</p>
<p><img src="/2019/10/09/Project-Summary/3.png" alt="1.png"></p>
<h3 id="HER2-enriched"><a href="#HER2-enriched" class="headerlink" title="HER2-enriched"></a>HER2-enriched</h3><table>
<thead>
<tr>
<th>Category</th>
<th>Term</th>
<th>Count</th>
<th>%</th>
<th>P Value</th>
</tr>
</thead>
<tbody><tr>
<td>KEGG_PATHWAY</td>
<td>hsa04152:AMPK signaling pathway</td>
<td>6</td>
<td>1.357466063</td>
<td>0.067021086</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa04914:Progesterone-mediated oocyte maturation</td>
<td>5</td>
<td>1.131221719</td>
<td>0.068221006</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa04922:Glucagon signaling pathway</td>
<td>5</td>
<td>1.131221719</td>
<td>0.098540503</td>
</tr>
</tbody></table>
<p>Ubiquitin mediated proteolysis</p>
<p><img src="/2019/10/09/Project-Summary/4.png" alt="1.png"></p>
<p>Amino sugar and nucleotide sugar metabolism</p>
<p><img src="/2019/10/09/Project-Summary/5.png" alt="1.png"></p>
<p>Sphingolipid signaling pathway</p>
<p><img src="/2019/10/09/Project-Summary/6.png" alt="1.png"></p>
<h3 id="luminal-b"><a href="#luminal-b" class="headerlink" title="luminal b"></a>luminal b</h3><table>
<thead>
<tr>
<th>Category</th>
<th>Term</th>
<th>Count</th>
<th>%</th>
<th>P Value</th>
</tr>
</thead>
<tbody><tr>
<td>KEGG_PATHWAY</td>
<td>hsa03420:Nucleotide excision repair</td>
<td>4</td>
<td>0.902934537</td>
<td>0.066436908</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa00534:Glycosaminoglycan biosynthesis - heparan sulfate / heparin</td>
<td>3</td>
<td>0.677200903</td>
<td>0.081737407</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa01100:Metabolic pathways</td>
<td>32</td>
<td>7.223476298</td>
<td>0.082719307</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa00563:Glycosylphosphatidylinositol(GPI)-anchor biosynthesis</td>
<td>3</td>
<td>0.677200903</td>
<td>0.087723263</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa03460:Fanconi anemia pathway</td>
<td>4</td>
<td>0.902934537</td>
<td>0.088216207</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>BIOCARTA</td>
<td>h_cptPathway:Mitochondrial Carnitine Palmitoyltransferase (CPT) System</td>
<td>3</td>
<td>0.677200903</td>
<td>0.002064389</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>BIOCARTA</td>
<td>h_mapkPathway:MAPKinase Signaling Pathway</td>
<td>5</td>
<td>1.128668172</td>
<td>0.079931529</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>BBID</td>
<td>103.Ca2-calmodulin_PK_cascade</td>
<td>2</td>
<td>0.451467269</td>
<td>0.068157821</td>
</tr>
</tbody></table>
<p>Mitochondrial Carnitine Palmitoyltransferase system </p>
<p><img src="/2019/10/09/Project-Summary/7.png" alt="1.png"></p>
<h3 id="K-Means-clustering-k-3"><a href="#K-Means-clustering-k-3" class="headerlink" title="K-Means clustering k=3"></a><center>K-Means clustering k=3</center></h3><p>cluster 1</p>
<table>
<thead>
<tr>
<th>Category</th>
<th>Term</th>
<th>Count</th>
<th>%</th>
<th>P Value</th>
</tr>
</thead>
<tbody><tr>
<td>KEGG_PATHWAY</td>
<td>hsa03420:Nucleotide excision repair</td>
<td>4</td>
<td>0.902934537</td>
<td>0.066436908</td>
</tr>
<tr>
<td>Category</td>
<td>Term</td>
<td>Count</td>
<td>%</td>
<td>PValue</td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa04931:Insulin resistance</td>
<td>8</td>
<td>1.758241758</td>
<td>0.007306318</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa04920:Adipocytokine signaling pathway</td>
<td>6</td>
<td>1.318681319</td>
<td>0.015269368</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa04261:Adrenergic signaling in cardiomyocytes</td>
<td>7</td>
<td>1.538461538</td>
<td>0.069049425</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa05410:Hypertrophic cardiomyopathy (HCM)</td>
<td>5</td>
<td>1.098901099</td>
<td>0.080224349</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa05414:Dilated cardiomyopathy</td>
<td>5</td>
<td>1.098901099</td>
<td>0.098845332</td>
</tr>
</tbody></table>
<p><img src="/2019/10/09/Project-Summary/8.png" alt="1.png"></p>
<p>Adipocytokine signaling pathway </p>
<p>![1.png]Project-Summary/9.png)</p>
<p>cluster 2</p>
<table>
<thead>
<tr>
<th>Category</th>
<th>Term</th>
<th>Count</th>
<th>%</th>
<th>PValue</th>
</tr>
</thead>
<tbody><tr>
<td>KEGG_PATHWAY</td>
<td>hsa00534:Glycosaminoglycan biosynthesis - heparan sulfate / heparin</td>
<td>4</td>
<td>0.883002208</td>
<td>0.015234368</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa04015:Rap1 signaling pathway</td>
<td>10</td>
<td>2.207505519</td>
<td>0.042576846</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa04115:p53 signaling pathway</td>
<td>5</td>
<td>1.103752759</td>
<td>0.060203345</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa05202:Transcriptional misregulation in cancer</td>
<td>8</td>
<td>1.766004415</td>
<td>0.076194837</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa03018:RNA degradation</td>
<td>5</td>
<td>1.103752759</td>
<td>0.090107337</td>
</tr>
</tbody></table>
<p>Glycosaminoglycan biosynthesis - heparin sulfate </p>
<p><img src="/2019/10/09/Project-Summary/10.png" alt="1.png"></p>
<p>RAP1 signaling pathway </p>
<p><img src="/2019/10/09/Project-Summary/11.png" alt="1.png"></p>
<p>cluster 3</p>
<table>
<thead>
<tr>
<th>Category</th>
<th>Term</th>
<th>Count</th>
<th>%</th>
<th>PValue</th>
</tr>
</thead>
<tbody><tr>
<td>KEGG_PATHWAY</td>
<td>hsa05202:Transcriptional misregulation in cancer</td>
<td>8</td>
<td>1.77383592</td>
<td>0.058792465</td>
</tr>
<tr>
<td>KEGG_PATHWAY</td>
<td>hsa00900:Terpenoid backbone biosynthesis</td>
<td>3</td>
<td>0.66518847</td>
<td>0.075613479</td>
</tr>
</tbody></table>
<p>Biological pathway: Here we only focus on the default pathways identifies by DAVID, which KEGG_PATHWAY, BIOCARTA, and BBID. And pathways with a P-Value lower than 0.05 are considered significant.</p>
<p>Gene differential analysis by PAM50 and K-Means clustering identifies very different biological pathways. Note there is no pathways with a P-Value lower than 0.05 in PAM50 subgroup HER2-enriched and in K-Means clustering subgroup cluster 3, so no pathways is considered to contribute to the uniqueness of HER2-enriched subtype and cluster 3 in this sample. Also notice that only a few pathways are identified in cluster 3 by K-Means clustering. This may be due to the fact that pathways other than KEGG-pathway are discarded in this project. Interestingly, in cluster 1, the insulin resistance pathway has a significantly low P-Value, which suggests it has a high possibility to be contributing to the uniqueness of cluster 1 and the diversity of breast cancer. Furthermore, in PAM50 subtypes, Basal-like and Luminal A share the same significant pathway —— amino sugar and nucleotide sugar metabolism, which might indicate that this pathway is correlated with HER2 negative breast cancer subtypes.</p>
</div></div><div class="footer"><div class="footer-copyright">Theme By <a href="https://github.com/shinux/hexo-theme-adoubi">Adoubi</a> , Powered By Hexo.</div></div></body></html>